llvm/include/llvm/IR/ModuleSummaryIndex.h

```diff
@@ -1291,6 +1291,9 @@ using ModulePathStringTableTy = StringMap<ModuleHash>;
 /// a particular module, and provide efficient access to their summary.
 using GVSummaryMapTy = DenseMap<GlobalValue::GUID, GlobalValueSummary *>;
 
+/// Map of a module name to its summary.
```

Comment:
s/summary/summaries/. And more accurately, map of a module name to the GUIDs and summaries we will import from that module.

Reply:
Updated.  Thanks!